### DISCUSSION TITLE
Can't add team project

### DISCUSSION THREAD

**Discussion by oleksandr-petryk** [community/community#32553] [Projects and Issues]
My organization has several `Public` projects, but I can't add them to my organization team. Can someone help me?

<img alt="Screenshot" src="https://user-images.githubusercontent.com/57829569/189483332-10f66c9c-aeaf-49ef-b6b0-7587aa1183c3.png">

**Comment by jfullstackdev** [comment]
seems this is fixed by now ? 

![image](https://user-images.githubusercontent.com/47092464/189490615-cbde19c6-d3ed-4de3-aa5d-6442913e6234.png)

**Reply by oleksandr-petryk** [reply]
It's organizaton projects, but I need to add them to the [team](https://github.com/orgs/Alerki/teams/website/discussions)
